### PR TITLE
Update package.json in React Integration for React 19

### PR DESCRIPTION
**Pull request by bskimball**
## Changes

Update peer dependencies with React Integration now that React 19 is stable

## Testing

Tested locally but React 19 beta was already supported. Stable version should also work the same

## Docs

These will prevent users from having to set the --peer-legacy-deps flag on npm install/uninstall

### CODE REVIEW

**Review by ematipico — CHANGES_REQUESTED** [review]
Two important things:
- the beta tag should be removed 
- we need to provide a changeset (it's a minor)

File: packages/integrations/react/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@astrojs/react",
   "description": "Use React components within Astro",
-  "version": "4.0.0",
+  "version": "4.1.0",
```
Comment (ematipico) [inline packages/integrations/react/package.json:4]: There's no need to update the version. That will be done automatically by changesets. You can revert the change

**Comment by bskimball** [comment]
I've messed this up with the changeset. I'll close and submit a new PR
